src/gui/search.ts

```diff
@@ -9,7 +9,7 @@
 
```

Comment:
**Regarding the whole file:** This file doesn't seem to be imported anywhere, since there is also an identical file in `src/doc/`.